operations/jasper.go

```diff
@@ -0,0 +1,104 @@
+package operations
+
+import (
+	"context"
+	"fmt"
+	"net"
+
+	"github.com/mongodb/grip"
+	"github.com/mongodb/jasper"
+	"github.com/mongodb/jasper/jrpc"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+	"google.golang.org/grpc"
+)
+
+func Jasper() cli.Command {
+	return cli.Command{
+		Name:  "jasper",
+		Usage: "tools for running jasper process management services",
+		Flags: []cli.Flag{},
+		Subcommands: []cli.Command{
+			jasperGRPC(),
+			jasperREST(),
+		},
+	}
+}
+
+func jasperGRPC() cli.Command {
+	return cli.Command{
+		Name:  "grpc",
+		Usage: "run jasper service accessible with gRPC",
+		Flags: []cli.Flag{
+			cli.IntFlag{
+				Name:   "port",
+				EnvVar: "JASPER_GRPC_PORT",
+				Value:  2286,
+			},
+		},
+		Action: func(c *cli.Context) error {
+			port := c.Int("port")
+			ctx, cancel := context.WithCancel(context.Background())
+			defer cancel()
```

Comment:
I don't think you want this. Since this function starts a goroutine which listens for done, deferring canceling means that when this function returns nil, it will then cancel the context. Also, if this is called at the CLI, it will exit immediately, which will also stop the serving goroutine. What blocks to keep the server up?